lib/pages/chat_list/chat_list_view.dart

```diff
@@ -68,7 +70,12 @@ class ChatListView extends StatelessWidget {
           }
         },
       ),
-      body: ChatListBodyView(controller),
+      body: StreamBuilder<Client>(
+        stream: controller.clientStream,
+        builder: (context, snapshot) {
+          return ChatListBodyView(controller);
+        },
+      ),
```

Comment:
why need this?

Reply:
No need, I'll remove it

Reply:
Trigger rebuild chat list when add new account